### PR TITLE
Lesson_1_7

### PR DESCRIPTION
**Pull request by AE-BHH**
Week 8:
Lesson_1_7
Asynchronous Data
Conditional Rendering
Advanced & Impossible State

### CODE REVIEW

File: src/App.js
Comment (JoHeyJo) [inline src/App.js]: `JSON.parse(localStorage.getItem('savedTodoList')` is already being applied on mount. Is this necessary?

File: src/App.js
Comment (JoHeyJo) [inline src/App.js:9]: This is fantastic! Good job also including error handling!
 Let's clean up all those console logs.

File: src/App.js
Comment (JoHeyJo) [inline src/App.js:34]: Nicely done implementing `removeTodo`.

**Review by JoHeyJo — CHANGES_REQUESTED** [review]
Well done implementing everything you've learned in the lesson, it wasn't an easy one. There are a few minor things to address but there is still a pretty big bug lingering from a previous lesson that somehow made it over to this branch. Your form still doesn't submit when you hit 'enter' on your keyboard. Please fix this before merging and moving on. 

Let's get everything sorted out before merging. I'll approve after revisions have been made.

File: src/App.js
Comment (AE-BHH) [reply src/App.js]: I removed this JSON.parse() method from todoList useState.
Thanks for the feedback!

File: src/App.js
Comment (AE-BHH) [reply src/App.js:9]: Yes, sure!

**Pull request merged**
